### PR TITLE
AB#90381 Update for Local Authority and Planning Status

### PR DESCRIPTION
**Pull request by dawid-pawulski**

### CODE REVIEW

File: src/HE.Investment.AHP.Contract/Site/Queries/GetLocalAuthorityQuery.cs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+using MediatR;
+
+namespace HE.Investment.AHP.Contract.Site.Queries;
+
+public record GetLocalAuthorityQuery(string LocalAuthorityId) : IRequest<LocalAuthority>;
```
Comment (jjankowski87) [inline src/HE.Investment.AHP.Contract/Site/Queries/GetLocalAuthorityQuery.cs:5]: Maybe we want to use StringId value object for Local Authority Id?

File: src/HE.Investments.Organisation/LocalAuthorities/Repositories/LocalAuthorityRepository.cs [hunk 2]
```diff
@@ -46,6 +46,19 @@ public async Task<(IList<LocalAuthority> Items, int TotalItems)> Search(string p
         return (itemsAtPage.ToList(), totalItems);
     }
 
+    public async Task<LocalAuthority> GetById(string localAuthorityId, CancellationToken cancellationToken)
+    {
+        var localAuthorities = await _cacheService.GetValueAsync(
```
Comment (jjankowski87) [inline src/HE.Investments.Organisation/LocalAuthorities/Repositories/LocalAuthorityRepository.cs:51]: this is the same code as in line 27, so should be extracted to private method

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investment.AHP.Contract/Site/Queries/GetLocalAuthorityQuery.cs (hunk 1 above)
Comment (dawid-pawulski) [reply src/HE.Investment.AHP.Contract/Site/Queries/GetLocalAuthorityQuery.cs]: nice, done

File: src/HE.Investments.Organisation/LocalAuthorities/Repositories/LocalAuthorityRepository.cs (hunk 2 above)
Comment (dawid-pawulski) [reply src/HE.Investments.Organisation/LocalAuthorities/Repositories/LocalAuthorityRepository.cs]: nice, done

**Review by Antek012 — APPROVED** [review]

**Pull request merged**